scripts/zones/RuLude_Gardens/npcs/Harith.lua

```diff
@@ -99,7 +99,13 @@ function onEventFinish(player,csid,option)
         player:tradeComplete();
         player:addItem(objecttrade);
         player:messageSpecial(ITEM_OBTAINED,objecttrade);
-        player:setVar("harithreward",0);
+        player:setVar("harithreward", 0);
+		if (player:getQuestStatus(JEUNO, EMPTY_MEMORIES) == QUEST_ACCEPTED) then
```

Comment:
You have tabs. Please configure your editor to automatically create 4 spaces when you press the tab-key\*

This project uses 4 spaces per indent level, so 2 indents would be 8 spaces between the 2.

##### \*Nobody actually sits there counting spaces, we just have the editor convert